api/src/test/resources/io/strimzi/api/kafka/model/Kafka-with-extra-property.yaml

```diff
@@ -23,7 +23,7 @@ spec:
       -Xmx: "4G"
       -Xms: "2G"
       -XX:
-        -foo: bar
+        foo: bar
```

Comment:
So if this was wrong the whole time, why do the tests not crash? Should there been some failing tests or something?

Reply:
The test was not wrong... just the property was called `-foo` not `foo` (as it was intended I believe)

Reply:
Ahh, ok. Uderstand now. Thanks